pkg/status/status.go

```diff
@@ -197,17 +182,20 @@ func (s *status) ensureClusterOperator() error {
 
 // setStatus handles setting all the required fields for the given
 // ClusterStatusConditionType
-func (s *status) setStatus(condition configv1.ClusterStatusConditionType, message string) error {
+func (s *status) setStatus(statusConditions []configv1.ClusterOperatorStatusCondition) error {
 	if s.coAPINotPresent {
-		return nil
+		return fmt.Errorf("ClusterOperator API not present")
```

Comment:
Why is this an error? Won't it break vanilla kube?

Reply:
`setStatus` is only called in the `monitorClusterStatus` routine which [isn't started on vanilla](https://github.com/operator-framework/operator-marketplace/blob/4fec4b65c1693eedc49f0f37fe13121fee59dfb8/pkg/status/status.go#L93). This would handle the usecase where the `CO API` was present on startup but was removed.